tests/tests_insights_remediation.yml

```
@@ -41,14 +42,18 @@
               insecure: "{{ lsr_rhc_test_data.candlepin_insecure }}"
             rhc_baseurl: "{{ lsr_rhc_test_data.baseurl | d(omit) }}"
 
+        - name: Set the fact for the systemd service
+          set_fact:
+            yggdrasil_systemd_service: "{{ __rhc_yggdrasil_systemd_service + '.service' }}"
+
         - name: Get service_facts
           service_facts:
 
         - name: Check remediation is enabled
           assert:
             that:
-              - "'rhcd.service' in ansible_facts.services"
-              - ansible_facts.services['rhcd.service'].status == 'enabled'
+              - "yggdrasil_systemd_service in ansible_facts.services"
```

```suggestion
              - yggdrasil_systemd_service in ansible_facts.services
```
This shouldn't be quoted anymore since there are no quoted strings being used anymore

---

ooh nice, thanks